jubilant/_juju.py

```diff
@@ -290,6 +290,32 @@ def destroy_model(
         if model == self.model:
             self.model = None
 
+    def integrate(self, app1: str, app2: str, *, via: str | Iterable[str] | None = None) -> None:
+        """Integrate two applications, creating a relation between them.
+
+        The order of *app1* and *app2* is not significant. Each of them should
+        be in the format ``<application>[:<endpoint>]``. The endpoint is only
```

Comment:
Is it worth saying that one of them can also be an offer URL or alias, either here or down in the CMR paragraph?

Reply:
Yeah, good idea. I've added a sentence about that, with "See ``juju integrate --help`` for details." as we don't want to repeat that whole thing here.